src/plugins/intel_cpu/tests/functional/subgraph_tests/src/ngram.cpp

```diff
@@ -207,7 +207,7 @@ class NgramCPUTest : public testing::WithParamInterface<NgramTestParams>, virtua
 
 TEST_P(NgramCPUTest, CompareWithRefs) {
     run();
-    CheckNumberOfNodesWithType(compiledModel, "Ngram", 1);
```

Comment:
Could you please explain why the number has been changed?

Reply:
Ngram is internal node and it's fused from graph based on some pattern. After tokenization, the pattern can not match, so there is no ngram node. Maybe I need disable tokenization for this case, it's intended for ngram test. Thank you Alexandra.

Reply:
I'd suggest to disable Snippets here to allow test to validate `Ngram` implementation. To validate subgraph we have eltwise tests and snippets tests. You can disable them using config (like [here](https://github.com/openvinotoolkit/openvino/blob/52b9df4a6d7cf80833c0e441c11fb80f12d6bfce/src/tests/functional/plugin/shared/src/snippets/mha.cpp#L58) but with `DISABLE` key). Thank you!

Reply:
done, thank you!